templates_ark/Segment_Image_Data.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
<p>Does this need to be updated?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/528/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
There were some imports that were unused in this notebook, specifically `load_utils`, and `io_utils` was imported twice.

Reply:
The error message about index out of range I mean, is something off in the notebook?

Reply:
<p>No, that was because I didn't install ark-analysis into my environment after I made the changes. After installing ark-analysis it worked fine.</p>